docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.7"
+version: "3.3"
```

Comment:
why reduce the version?

Reply:
Gave an error on my machine for version 3.7, worked with 3.3